### ISSUE TITLE
Error: Request textDocument/codeAction failed. invalid value: string "[object Object]", expected relative URL without a base

### ISSUE THREAD

**Issue by evelant** [ast-grep/ast-grep-vscode#314]
Not sure what changed but anytime I restart vscode this error pops up several times.

```
Parsing changed doc.
Publishing diagnostics.
[Error - 2:25:26 PM] Request textDocument/codeAction failed.
  Message: invalid value: string "[object Object]", expected relative URL without a base
  Code: -32602
```

**Comment by HerringtonDarkholme** [comment]
Do you have a reproduction repo? also what's your vscode version? have you disabled other extensions?

**Comment by HerringtonDarkholme** [comment]
I believe thsi should be fixed in ast-grep 0.25.2, closing now.

If you still find the issue persistent, please kindly file an issue with reproduction.
